.github/workflows/deploy_documentation.yml

```diff
@@ -3,12 +3,12 @@ name: Deploy Documentation
 # Controls when the action will run. Triggers the workflow on push
 on:
   workflow_dispatch:
-  push:
+  pull_request:
     branches: [ master ]
 
-    paths:
-    - 'sssom_schema/src/docs/*'
-    - 'sssom_schema/src/linkml/sssom_schema.yaml'
+    # paths:
```

Comment:
Why are these commented out? These are just conditions for when the action should be triggered, right?